src/Service/Form/Submission/FormSubmissionRequest.php

```diff
@@ -41,7 +41,7 @@ public function __construct(Request $request)
         $this->data = $submit['data'];
         $this->files = $submit['files'];
         $this->label = $submit['label'] ?? '';
-        $formattedDate = \DateTime::createFromFormat('c', $submit['expire_date']);
+        $formattedDate = \DateTime::createFromFormat(\DateTime::ISO8601, $submit['expire_date']);
```

Comment:
Note: This format is not compatible with ISO-8601, but is left this way for backward compatibility reasons. Use DateTime::ATOM or DATE_ATOM for compatibility with ISO-8601 instead.